src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style)
+    .reduce((stylesObject, style) => {
+      const [property, value] = style.split(':').map((part) => part.trim());
+
+      if (property && value) {
+        stylesObject[property] = value;
+      }
+
+      return stylesObject;
+    }, {});
 }
```

Comment:
The function `convertToObject` is well-implemented using a chain of string and array methods. However, the checklist specifies to avoid using loops and to prefer iteration methods. The `reduce` method here is acceptable as it is being used to construct a new object from an array, which is a common use case for `reduce`.